events/wc_vrykul_events.txt

```diff
@@ -1 +1,693 @@
+
+	is_triggered_only = yes
+
+	major = yes
+	major_trigger = {
+		ai = no
+	}
+
+	immediate = {
+		log = "WCTHW.130 fired for [Root.GetBestName]"
+		
+		set_global_flag = drust_incursion_happened
+		
+		create_character = {
+			name = Gorak
+			dynasty = 70201
+			culture = drust
+			religion = throsic
+			attributes = {
+				martial=5 diplomacy=5 stewardship=5 intrigue=6 learning=6
+			}
+			trait=mastermind_theologian
+			trait=creature_vrykul
+			trait=class_necromancer_9
+			trait=cruel trait=patient trait=zealous trait=deceitful trait=being_undead
+			age = 500
+		}
+		new_character = {
+			save_event_target_as = target_gorak_tul
+			# Used in scripted effects
+			save_event_target_as = target_settler
+
+			# effect = {
+				# set_graphical_culture = vrykul_undead		# Nani is that? There must be no effect!
+			# }
+			
+			set_graphical_culture = vrykul_undead
+			create_bloodline = {
+				type = gorak_tul
+				has_dlc = "Holy Fury"
+			}
+			
+			c_fates_end = {
+				# event_target:target_settler and its people settle here	# without gameover if current ruler is player
+				settle_as_target_settler_effect = yes
+			}
+			
+			clear_settler_flag_effect = yes
+			
+			activate_title = { title = k_drustvar status = yes }
+			k_drustvar = { grant_title = event_target:target_gorak_tul }
+			
+			wealth = 1000
+			set_government_type = tribal_monarchy_government
+			
+			spawn_unit = {
+				owner = event_target:target_gorak_tul
+				province = 3553
+				home = 3553
+				troops = {
+					archers = { 2000 2000 }
+					light_infantry = { 3000 3000 }
+					heavy_infantry = { 3750 3750 }
+					light_cavalry = { 500 500 }
+					knights = { 750 750 }
+				}
+				cannot_inherit = no
+				attrition = 1.0
+				reinforces = yes
+				maintenance = no
+			}
+			spawn_fleet = {
+				province = closest # closest sea zone
+				owner = event_target:target_gorak_tul
+				disband_on_peace = yes
+				troops = {
+					galleys = { 100 100 }
+				}
+			}
+			
+			create_character = {
+				name = Maloch
+				dynasty = 70202
+				culture = drust religion = throsic
+				attributes = {
+					martial=5 diplomacy=4 stewardship=5 intrigue=4 learning=4
+				}
+				trait=mastermind_theologian
+				trait=creature_vrykul
+				trait=class_necromancer_5
+				trait=being_undead trait=zealous trait=cruel trait=proud
+				age = 400
+				set_graphical_culture = vrykul_undead
+			}
+			
+			create_character = {
+				name = Drif
+				dynasty = 70203
+				culture = drust religion = throsic
+				attributes = {
+					martial=6 diplomacy=3 stewardship=4 intrigue=5 learning=4
+				}
+				trait=brilliant_strategist
+				trait=creature_vrykul
+				trait=class_warrior_5
+				trait=being_undead trait=zealous trait=sturdy trait=brave trait=wroth
+				age = 400
+				set_graphical_culture = vrykul_undead
+			}
+		}
+		# Make the person that fired the decison a vassal of Gorak Tul
+		event_target:target_summoner_of_gorak = {
+			set_defacto_liege = event_target:target_gorak_tul
+		}
```

Comment:
You should check `character_can_be_vassalized_trigger` here.

Reply:
pls verify if correct

Reply:
LGTM!